tests/bdd/test_csi_controller.py

```diff
@@ -596,6 +663,13 @@ def _create_1_replica_nvmf_volume():
     csi_delete_1_replica_nvmf_volume1()
 
 
+@pytest.fixture
+def _create_1_replica_local_nvmf_volume():
+    csi_delete_1_replica_local_nvmf_volume1()
```

Comment:
Why do we need to delete it first? We will do this as part of the cleanup after the `yield` below won't we?